lib/tasks/schedule_rules.rake

```diff
@@ -20,4 +23,34 @@ namespace :schedule_rule do
       end
     end
   end
+
+  # Add missing global price groups to schedule rules.
+  #
+  # Run this whenever a new global price group is created. You can set a discount
+  # percent for all schedule rules. The below example will set a 20% discount
+  #
+  # `rake schedule_rule:add_new_price_groups[20]`
+  desc "Adds PriceGroupDiscounts for global price groups that have yet to be associated with schedule rules"
+  task :add_new_price_groups, [:discount] => :environment do |_t, args|
```

Comment:
- Move to new file, price_groups.rake
- Add relevant documentation in that file
- Task accepts a pg_name, is_internal, admin_editable, and discount
- Task uses a new class method PriceGroup.new_global